scaleway/resource_iam_api_key_test.go

```diff
@@ -55,7 +55,7 @@ func TestAccScalewayIamApiKey_WithApplication(t *testing.T) {
 			{
 				Config: `
 						resource "scaleway_iam_application" "main" {
-							name = "tf_tests_app_basic"
+							name = "tf_tests_app_key_basic"
```

Comment:
Why changing a name will fix insufficient permissions? https://github.com/scaleway/terraform-provider-scaleway/actions/runs/4059836862/jobs/6988323260#step:4:7929

Reply:
fixes not linked to this